### PR TITLE
refactor(cdk): drop unused scheduler APIs

### PR DESCRIPTION
**Pull request by edbzn**
Clean-up unused scheduler APIs, only `forceFramerate`, `scheduleCallback`, and `cancelCallback` are effectively used.

### CODE REVIEW

File: libs/cdk/internals/scheduler/src/lib/scheduler.ts [hunk 1]
```diff
@@ -71,18 +67,10 @@ const clearTimeout = ɵglobal.clearTimeout;
...
 
-const isInputPending =
-  typeof ɵglobal.navigator !== 'undefined' &&
-  ɵglobal.navigator.scheduling !== undefined &&
-  ɵglobal.navigator.scheduling.isInputPending !== undefined
-    ? ɵglobal.navigator.scheduling.isInputPending.bind(
-        ɵglobal.navigator.scheduling
-      )
-    : null;
-
```
Comment (hoebbelsB) [inline libs/cdk/internals/scheduler/src/lib/scheduler.ts:82]: this is actually something we want to keep.

File: libs/cdk/internals/scheduler/src/lib/scheduler.ts [hunk 2]
```diff
@@ -364,33 +289,13 @@ function scheduleCallback(
...
-  isSchedulerPaused = true;
-}
-
-function continueExecution() {
-  isSchedulerPaused = false;
-  if (!isHostCallbackScheduled && !isPerformingWork) {
-    isHostCallbackScheduled = true;
-    requestHostCallback(flushWork);
-  }
-}
```
Comment (hoebbelsB) [inline libs/cdk/internals/scheduler/src/lib/scheduler.ts:377]: same goes for them, I guess they are connected

**Review by hoebbelsB — CHANGES_REQUESTED** [review]
hey @edbzn could you restore the things we want to keep and rebase?

File: libs/cdk/internals/scheduler/src/lib/scheduler.ts (hunk 1 above)
Comment (edbzn) [reply libs/cdk/internals/scheduler/src/lib/scheduler.ts:82]: Do we want to keep something not used? Why?

File: libs/cdk/internals/scheduler/src/lib/scheduler.ts (hunk 2 above)
Comment (edbzn) [reply libs/cdk/internals/scheduler/src/lib/scheduler.ts:377]: Same question.

File: libs/cdk/internals/scheduler/src/lib/scheduler.ts (hunk 1 above)
Comment (hoebbelsB) [reply libs/cdk/internals/scheduler/src/lib/scheduler.ts:82]: because it is part of the roadmap to test this API :D

Comment (edbzn) [reply libs/cdk/internals/scheduler/src/lib/scheduler.ts:82]: Then we could introduce it when we need it.

Comment (hoebbelsB) [reply libs/cdk/internals/scheduler/src/lib/scheduler.ts:82]: okay, fine by me

**Comment by hoebbelsB** [comment]
> hey @edbzn could you restore the things we want to keep and rebase?

can u just rebase then? :D

**Pull request merged**